rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala

```diff
@@ -149,6 +160,7 @@ class ReportsExecutionService (
         reportsRepository.getReportsWithLowestId match {
           case Full(Some((id, report))) =>
             logger.debug(s"Initializing the status execution update to  id ${id}, date ${report.executionTimestamp}")
+            this.idForCheck = id
```

Comment:
I would understand if the 'this.' was on 'id', but I don't understand why you need to add it to IdForCheck

```idForCheck = id ```

Would have been what I expect!